docs/ref/reference.rst

```diff
@@ -125,10 +133,14 @@ commands are dealing with the monitor:
      DNS lookup based process and force the local node name to a fixed
      value.
      
-     The ``--auth`` option allows setting up authentication method to be used
-     for connections from data nodes with ``autoctl_node`` user.
-     If this option is used, please make sure password is manually set on
-     the monitor, and appropriate setting is added to `.pgpass` file on data node.
+     The ``--auth`` option allows setting up authentication method to be
+     used for connections from data nodes with ``autoctl_node`` user. When
+     testing pg_auto_failover for the first time using ``--auth trust``
+     makes things easier. When getting production ready, review your options
+     here and choose at least ``--auth scram-sha-256`` and make sure
+     password is manually set on the monitor, and appropriate setting is
+     added to `.pgpass` file on data node. You could also use some of the
+     advanced Postgres authentication mechanism such as SSL certificates.
```

Comment:
we should mention if we use either of  --auth or --skip-pg-hba options we have to setup autoctl_node user in the monitor and setup client access mechanism before creating any pg_autoctl node. 

Otherwise whole system might be in an unstable state.  pg_autoctl create succeeds without registering the node to the monitor. following pg_autoctl run command fails because the node is node registered

Reply:
I can't reproduce the problem locally anymore. But  still believe we mention it